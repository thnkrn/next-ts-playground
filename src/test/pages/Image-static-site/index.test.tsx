import ImageStaticSite from 'pages/Image-static-site'
import { render } from 'test/util'

describe('ImageStaticSite', () => {
  describe('Rendering', () => {
    it('should match snapshot', () => {
      const { asFragment } = render(<ImageStaticSite />)

      expect(asFragment()).toMatchSnapshot()
    })
  })
})
