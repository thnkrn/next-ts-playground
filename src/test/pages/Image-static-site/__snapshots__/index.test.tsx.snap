// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageStaticSite Rendering should match snapshot 1`] = `
<DocumentFragment>
  <style
    data-cmpld="true"
  >
    ._19pk1wug{margin-top:auto}._2hwx1wug{margin-right:auto}._otyr1wug{margin-bottom:auto}._18u01wug{margin-left:auto}._1bsb1n8h{width:70%}@media screen and (max-width:480px){._18l813sl{width:85%}}
  </style>
  <div
    class="_19pk1wug _2hwx1wug _otyr1wug _18u01wug _1bsb1n8h _18l813sl"
  >
    <h1
      class="h1"
    >
      Image Static Site
    </h1>
  </div>
</DocumentFragment>
`;
